docs/welcome/what_is_osparc.md

```diff
@@ -3,7 +3,7 @@
 *osparc* will provide users with an interactive approach to effectively develop, extend, validate, certify, document, store, share, and apply models, explore the impact of stimulation parameters, and create predictive, multi-scale, multi-physics models for a wide range of scenarios.
 One of the core elements of the platform is a set of novel neuro- and physiology-functionalized human and animal anatomical models (called NEUROCOUPLE and NEUROFAUNA, respectively). The models will allow simulation of in vivo fields generated by implanted or external stimulators, elucidation of the resulting neuromodulations, and assessment of the changes induced in organ physiology. A brief introduction to the platform's functionality and use can be found here:
 
-[![webinar1](https://user-images.githubusercontent.com/28002886/86373988-54a49a80-bc84-11ea-9692-da7ee4c02604.PNG ':size=50%')](https://www.youtube.com/watch?v=PNeX2yCkzs4&t)
+[![webinar1](https://user-images.githubusercontent.com/28002886/86373988-54a49a80-bc84-11ea-9692-da7ee4c02604.PNG)](https://www.youtube.com/watch?v=PNeX2yCkzs4&t)
```

Comment:
Have you considered embedding it? With this code
```
<p align="center">
<iframe width="784" height="441" src="https://www.youtube.com/embed/PNeX2yCkzs4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</p>
```
Current is on the top and the new code is on the bottom
![Screen Shot 2020-07-03 at 13 20 59](https://user-images.githubusercontent.com/32402063/86464564-0c42b680-bd30-11ea-9879-18747158b894.png)

Reply:
I... would have considered it... if I knew that was an option =D Thanks Pedro, yes that sounds like an even better idea.